### DISCUSSION TITLE
how to activate the PII Direct-to-Consumer Delivery (Restricted)

### DISCUSSION THREAD

**Discussion by AtikSrl** [amzn/selling-partner-api-models#3061] [General]
hello,
to be able to develop a connector to our customer with which to bring Amazon orders directly into his ERP, we need to activate the PII Direct-to-Consumer Delivery (Restricted)

Amazon continuously rejects our requests whether made with the customer account or with our account

@LindaNC we have seen that you are very active in this area, can you help us on this thing? we can not understand what is not good in the data that we put in the request form that is promptly rejected

I hope you can help us
greetings
AtikSrl

**Comment by jrmaia** [comment]
Any solution?

**Comment by benpatel** [comment]
We are facing same issue.

**Comment by jaimezac27** [comment]
Hello,

I applied for PII 1 month ago. Was requested to me to fill a file with many questions. I completed it and submitted. There has been 3 weeks now and I dont get any answer. Please, can someone help me? The case is 11858543591.

Many thanks
